app/assets/stylesheets/cclow/_events_timeline.scss

```diff
@@ -17,37 +17,37 @@ $arrow-btn-width: 48px;
     position: relative;
     text-align: center;
     width: 275px;
-  }
-
-  .time-point.time-point-multiple-events {
-    &:first-child {
-      text-align: left;
-      width: 145px;
-    }
-  }
 
-  .time-point-multiple-events {
-    &:last-child {
-      .point {
-        &:after {
-          border: none;
-        }
+    &.time-point-multiple-events {
+      &:first-child {
+        text-align: left;
```

Comment:
this could be moved into first-child of &-multiple-events right?

Reply:
As I can see there is no need to check if both classes are on the element time-point and time-point-multiple-events, hmm